### ISSUE TITLE
Error message during chat

### ISSUE THREAD

**Issue by BlindingDark** [zevlg/telega.el#211]
## Telega Setup
*OS*: Manjaro Linux
*Emacs*: GNU Emacs 28.0.50 (build 1, x86_64-pc-linux-gnu, GTK+ Version 3.24.23, cairo version 1.17.3)
*Telega*: telega v0.6.30 (TDLib v1.6.9) (telega-server v0.6.6)
*MELPA*: 20201008.1332

## Current Behavior

`toggle-debug-on-error`

```
Debugger entered--Lisp error: (void-variable tracking-buffers)
  #f(compiled-function (event) #<bytecode 0x138ac9db2f823e11>)((:@type "updateChatReadInbox" :chat_id 697402248 :last_read_inbox_message_id 45958037504 :unread_count 0))
  telega--on-event((:@type "updateChatReadInbox" :chat_id 697402248 :last_read_inbox_message_id 45958037504 :unread_count 0))
  telega-server--dispatch-cmd("event" (:@type "updateChatReadInbox" :chat_id 697402248 :last_read_inbox_message_id 45958037504 :unread_count 0))
  apply(telega-server--dispatch-cmd ("event" (:@type "updateChatReadInbox" :chat_id 697402248 :last_read_inbox_message_id 45958037504 :unread_count 0)))
  telega-server--parse-commands()
  telega-server--filter(#<process telega-server> "event 105\n(:@type \"updateChatReadInbox\" :chat_id 6...")
```

## Steps to Reproduce

1. chat with someone
2. error appear after sending messages

**Comment by furiel** [comment]
I ran into this problem too. It seems `tracking` is a dependency for telega. I could workaround that problem by installing it separately:

```elisp
  (use-package tracking
    :ensure t)
```

Since then I do not see that error.

Probably `tracking` needs to be listed in:
https://github.com/zevlg/telega.el/blob/df7fb2b0b9233e1c63128071338af904d8fa7b14/telega.el#L9

**Comment by furiel** [comment]
Strangely, https://github.com/zevlg/telega.el/commit/92f51734bf5314150fb0c12745ef7ca7a111975a did not fix the problem for me.

I edited
```
.emacs.d/elpa/telega-20201008.1332/telega-tdlib-events.el
```
by adding the single line in https://github.com/zevlg/telega.el/commit/92f51734bf5314150fb0c12745ef7ca7a111975a.

Then I deleted the elc files: `rm *.elc`
and recompiled with `make compile`.

After restarting emacs, and trying to send a message, I still receive:

```
telega--on-event: Symbol’s value as variable is void: tracking-buffers
```
